pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -527,6 +552,115 @@ func (exp *flowExporter) addRecordToSet(record flowexporter.FlowRecord) error {
 	return nil
 }
 
+func (exp *flowExporter) addDenyConnToSet(conn *flowexporter.Connection) error {
+	nodeName, _ := env.GetNodeName()
+	exp.ipfixSet.ResetSet()
+	templateID := exp.templateIDv4
+	if conn.IsIPv6 {
+		templateID = exp.templateIDv6
+	}
+	if err := exp.ipfixSet.PrepareSet(ipfixentities.Data, templateID); err != nil {
+		return err
+	}
+	// Iterate over all infoElements in the list
+	eL := exp.elementsListv4
+	if conn.IsIPv6 {
+		eL = exp.elementsListv6
+	}
+	for _, ie := range eL {
+		switch ieName := ie.Element.Name; ieName {
+		case "flowStartSeconds":
+			ie.Value = uint32(conn.TimeSeen.Unix())
+		case "flowEndSeconds":
+			ie.Value = uint32(conn.TimeSeen.Unix())
```

Comment:
I am trying to understand how this works. If I have a UDP flow that is dropped because of a NetworkPolicy, I would expect that UDP flow to be handled in the same way as a regular connection by the exporter: the `flowStartSeconds` field should match the first packet in I observed for that flow and the `flowEndSeconds` field should match the most recent packet observed? And is `EndOfFlowReason` always the right value for `flowEndReason`?

Reply:
Makes sense to me. For flow reason I kept `ActiveTimeout` and `IdleTimeout` similar to what exporter does.